content/en/docs/tasks/configmap-secret/managing-secret-using-config-file.md

```diff
@@ -180,6 +181,6 @@ kubectl delete secret mysecret
 ## {{% heading "whatsnext" %}}
 
 - Read more about the [Secret concept](/docs/concepts/configuration/secret/)
-- Learn how to [manage Secrets with the `kubectl` command](/docs/tasks/configmap-secret/managing-secret-using-kubectl/)
+- Learn how to [Manage Secrets using kubectl](/docs/tasks/configmap-secret/managing-secret-using-kubectl/)
```

Comment:
I don't think that the capitalization is particularly required.

Reply:
I think it's not bad to keep the capitalization of article title as is. But I have no strong preference over two style, maybe it's up to the author.